### PR TITLE
Partial TL translations for client-tfs.asc - 1,425 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
Good day everyone! Here's a partially translated file. Please review my work. Thank you.

### CODE REVIEW

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+(((Pakikipag-ugnayan sa ibang mga VCS, TFS)))
 (((TFS)))((("TFVC", see="TFS")))
-Git is becoming popular with Windows developers, and if you're writing code on Windows, there's a good chance you're using Microsoft's Team Foundation Server (TFS).
-TFS is a collaboration suite that includes defect and work-item tracking, process support for Scrum and others, code review, and version control.
-There's a bit of confusion ahead: *TFS* is the server, which supports controlling source code using both Git and their own custom VCS, which they've dubbed *TFVC* (Team Foundation Version Control).
-Git support is a somewhat new feature for TFS (shipping with the 2013 version), so all of the tools that predate that refer to the version-control portion as ``TFS'', even though they're mostly working with TFVC.
+Ang Git ay nagiging popular sa mga developer ng Windows, at kung ikaw ay sumusulat ng code sa Windows, mayroong magandang posibilidad na ikaw ay gumagamit ng Team Foundation Server (TFS) ng Microsoft.
+Ang TFS ay isang kolaborasyon na suite na naglalaman ng kasiraan at work-item na pagmamasid, proseso na suporta para sa Scrum at iba pa, code review, at kontrol ng bersyon.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: suporta to sumusuporta

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
+Mayroong kakaunting kalituhan sa unahan: Ang *TFS* ay ang server, na sumusuporta sa pagkontrol ng source code gamit ang parehong Git at sarili nitong pasadyang VCS, na tinawag nilang *TFVC* (Team Foundation Version Control).
+Ang Git na suporta ay isang medyo bagong tampok para sa TFS (ipinapadala kasama ang 2013 na bersyon), kaya lahat ng mga kasangkapan na nauuna sa iyon na tumutukoy sa version-control na bahagi bilang ``TFS'', kahit na sila ay kadalasang nagtatrabaho gamit ang TFVC.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: version-control to bersyon-kontrol

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
 
-If you find yourself on a team that's using TFVC but you'd rather use Git as your version-control client, there's a project for you.
+Kung natagpuan mo ang iyong sarili sa isang koponan na gumagamit ng TFVC ngunit gugustuhin mong gumamit ng Git bilang iyong version-control na kliyente, may isang proyekto para sa iyo.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: version-control to bersyon-kontrol

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
-This does mean you'll need access to those assemblies, which means you need to install a recent version of Visual Studio (any edition since version 2010, including Express since version 2012), or the Visual Studio SDK.
+Ang Git-tfs (matatagpuan sa https://github.com/git-tfs/git-tfs[]) ay isang .NET na proyekto, at (batay sa pagkakasulat nito) ito ay tumatakbo lamang sa Windows.
+Upang magtrabaho gamit ang mga repositoryo ng Git, ito ay gumagamit ng .NET na mga ugnayan para sa libgit2, isang library-oriented na pagpapatupad ng Git na napakainam sa pagganap at nagpapahintulot ng maraming umaangkop sa bituka ng isang Git na repositoryo.
+Ang Libgit2 ay hindi isang kompletong pagpapatupad ng Git, kaya upang sakupin ang mga kaibahan ang git-tfs ay talagang tumatawag ng command-line ng Git na kliyente para sa ilang mga operasyon, kaya walang artipisyal na mga hangganan sa kung ano ang maaari nitong gawin sa Git na mga repositoryo.
+Ang suporta nito sa TFVC na mga tampok ay sobrang gulang, dahil ito ay gumagamit ng Visual Studio na mga assembly para sa mga operasyon sa mga server.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: assembly to pagpupulong

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 5]
```diff
@@ -1,54 +1,54 @@
...
+Ang Codeplex (https://www.codeplex.com[]) o Visual Studio Online (http://www.visualstudio.com[]) ay parehong magandang pagpipilian para rito.
 ====
 
 
-===== Getting Started: `git-tf`
+===== Pagsisimula: `git-tf`
 
-The first thing you do, just as with any Git project, is clone.
-Here's what that looks like with `git-tf`:
+Ang unang bagay na gagawin mo, katulad ng anumang Git na proyekt, ay mag-clone.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: proyekt to proyekto

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 6]
```diff
@@ -80,15 +80,15 @@ d44b17a (HEAD, tag: TFS_C35190, origin_tfs/tfs, master) Goodbye
         Team Project Creation Wizard
 ----
 
-Notice the tags with names like `TFS_C35189`; this is a feature that helps you know which Git commits are associated with TFVC changesets.
-This is a nice way to represent it, since you can see with a simple log command which of your commits is associated with a snapshot that also exists in TFVC.
-They aren't necessary (and in fact you can turn them off with `git config git-tf.tag false`) – git-tf keeps the real commit-changeset mappings in the `.git/git-tf` file.
+Pansinin ang mga tag na may mga pangalan katulad ng `TFS_C35189`; ito ay isang tampok na tumutulong sa iyo upang malaman kung anong Git na mga commit ang nauugnay sa TFVC na mga hanay ng pagbabago.
+Ito ay isang magandang paraan upang irepresenta ito, dahil maaari mong tingnan gamit ang isang simpleng log na utos kung ano ang iyong mga commit na nauugnay sa isang snapshot na umiiral din sa TFVC.
+Sila ay hindi na kinakailangan (at sa katunayan maaari mong i-off ang mga ito gamit ang `git config git-tf.tag false`) – ang git-tf ay pinapanatili ang tunay na commit-changeset na mga pag-map sa `.git/git-tf` na file.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: pag-map to pagmapa

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 7]
```diff
@@ -80,15 +80,15 @@ d44b17a (HEAD, tag: TFS_C35190, origin_tfs/tfs, master) Goodbye
...
 
 
-===== Getting Started: `git-tfs`
+===== Pagsisimula: `git-tfs`
 
-Git-tfs cloning behaves a bit differently.
-Observe:
+Ang Git-tfs na pag-clone ay nag-uugali ng medyo may kaibahan.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: nag-uugali to kumikilos

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 8]
```diff
@@ -128,19 +128,19 @@ Date:   Fri Aug 1 03:41:59 2014 +0000
     git-tfs-id: [https://username.visualstudio.com/DefaultCollection]$/myproject/Trunk;C16
 ----
 
-There are two local branches, `master` and `featureA`, which represent the initial starting point of the clone (`Trunk` in TFVC) and a child branch (`featureA` in TFVC).
-You can also see that the `tfs` ``remote'' has a couple of refs too: `default` and `featureA`, which represent TFVC branches.
-Git-tfs maps the branch you cloned from to `tfs/default`, and others get their own names.
+May dalawang lokal na mga branch, ang `master` at `featureA`, na nagrerepresenta ng paunang panimulang punto ng clone (`Trunk` sa TFVC) at isang child na branch (`featureA` sa TFVC).
+Maaari mo ring makita na ang `tfs` na ``remote'' ay mayroon ding pares ng refs: ang `default` at `featureA`, na kumakatawan sa TFVC na mga branch.
+Ang Git-tfs ay magma-map ng branch na kung saan ka nag-clone patungo sa `tfs/default`, at ang mga iba pa ay kukuha ng kanilang sariling mga pangalan.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: magma-map to nagmamapa

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 9]
```diff
@@ -128,19 +128,19 @@ Date:   Fri Aug 1 03:41:59 2014 +0000
...
 
-Another thing to notice is the `git-tfs-id:` lines in the commit messages.
-Instead of tags, git-tfs uses these markers to relate TFVC changesets to Git commits.
-This has the implication that your Git commits will have a different SHA-1 hash before and after they have been pushed to TFVC.
+Iba pang bagay na pansinin ay ang `git-tfs-id:` na mga linya sa commit na mga mensahe.
+Sa halip na mga tag, ang git-tfs ay gumagamit ng mga marker na ito upang iugnay ang TFVC na mga hanay ng pagbabago sa Git na mga commit.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: marker  to marka

File: book/09-git-and-other-scms/sections/client-tfs.asc (hunk 8 above)
Comment (josteinfrancois) [reply book/09-git-and-other-scms/sections/client-tfs.asc]: will use magmamapa instead

**Comment by josteinfrancois** [comment]
Hey @robin-ho . Already did some updates thanks to your review.

File: book/09-git-and-other-scms/sections/client-tfs.asc (hunk 5 above)
Comment (severinolorillajr) [reply book/09-git-and-other-scms/sections/client-tfs.asc]: one last update @josteinfrancois and we can merge this.

Comment (josteinfrancois) [reply book/09-git-and-other-scms/sections/client-tfs.asc]: Changing it now

**Pull request merged**